### DISCUSSION TITLE
Allow Model binding to * attributes

### DISCUSSION THREAD

**Discussion by raimondsL** [livewire/livewire#4655] [3. Feature ideas]
I have a couple of JSON columns and casting them as array in my Model.

I would like to create a $rule = ['modelName.options.*'] so that I can bind directly to 'options' array and add both: value or key => value.

Livewire\Exceptions\CannotBindToModelDataWithoutValidationRuleException
Cannot bind property [modelName.options.socket] without a validation rule present in the [$rules] array on Livewire component:

Otherwise I have to store the values in array and then transfer them.

**Comment by joshhanley** [comment]
@raimondslinde yeah makes sense. I'd probably just store then in an array property and then transfer them for now.
